jupyter_sphinx/ast.py

```diff
@@ -45,8 +41,8 @@ def load_content(cell, location, logger):
         try:
             with Path(filename).open() as f:
                 content = [line.rstrip() for line in f.readlines()]
-        except (IOError, OSError):
-            raise IOError("File {} not found or reading it failed".format(filename))
+        except OSError:
+            raise OSError(f"File {filename} not found or reading it failed")
```

Comment:
Why the change to the exception types?

Reply:
This change is made by pyupgrade and is equivalent because we support Python≥3.6 only.

See https://github.com/asottile/pyupgrade#oserror-aliases

Reply:
`isinstance(IOError(), OSError) == True`